packages/analytics/amplify_analytics_pinpoint_dart/lib/src/impl/drift/drift_tables.dart

```diff
@@ -0,0 +1,84 @@
+// Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'package:amplify_analytics_pinpoint_dart/src/impl/drift/connection/connection.dart'
+    as impl;
+import 'package:amplify_analytics_pinpoint_dart/src/impl/flutter_provider_interfaces/cached_events_path_provider.dart';
+import 'package:drift/drift.dart';
+
+part 'drift_tables.g.dart';
+
+/// {@template amplify_analytics_pinpoint_dart.drift_json_strings}
```

Comment:
I am glad you are using templates, but you don't need to use a template if the comment it not reused. I don't see this template referenced anywhere. I think there are a handful of other templates in this PR that are also not used anywhere.

Reply:
Yeah I made sure to use them.  Did you see that below? 

**/// {@template amplify_analytics_pinpoint_dart.drift_database_json_strings}**
/// Drift class for managing stored [DriftJsonStrings]
/// {@endtemplate}
@DriftDatabase(tables: [DriftJsonStrings])
class DriftDatabaseJsonStrings extends _$DriftDatabaseJsonStrings {
  **/// {@macro amplify_analytics_pinpoint_dart.drift_database_json_strings}**
  DriftDatabaseJsonStrings({CachedEventsPathProvider? pathProvider})
      : super(impl.connect(pathProvider));

  static DriftDatabaseJsonStrings? _instance;

  **/// {@macro amplify_analytics_pinpoint_dart.drift_database_json_strings}**
  static DriftDatabaseJsonStrings getInstance({
    CachedEventsPathProvider? pathProvider,
  }) {
    return _instance ??= DriftDatabaseJsonStrings(
      pathProvider: pathProvider,
    );
  }